index.html

```diff
@@ -0,0 +1,96 @@
+                    </li>
+                    <li class="lineItems">
+                        <div class="searchBar">
+                            <i class="searchLogo"></i>
+                            <input id="searchText">
+                        </div>
+                    </li>
+                </ul>
+            </nav>
+        </div>
+        <div class="mainText">
+            <h1><a href="#" class="noStyle"> What will your verse be? </a> </h1>
+            <h2><a href="#" class="noStyle">Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story.</a></h2>
+            <div class="thirdText">
+                <h3><a href="#" class="noStyle"> Watch the "Your Verse" film and explore the stories </a> </h3>
+                <img src="images/play-button-28245.png" class="playimage" alt="Play Button">
+            </div>
+        </div>
+    </div>
+    <div class="images">
+        <ul class="imageList">
+            <li class="imagebelow">
+                <img src="images/1promo_iphone5s.jpg" alt="image1">
+            </li>
+            <li class="imagebelow">
+                <img src="images/2promo_iphone5c.jpg" alt="image2">
+            </li>
+            <li class="imagebelow">
+                <img src="images/3promo_30_years.jpg" alt="image3">
+            </li>
+            <li class="imagebelow">
+                <img src="images/4promo_supplier_responsibility.jpg" alt="image4">
+            </li>
+        </ul>
+    </div>
+    <footer>
+        <div class="footerFlex">
+            <div class="footerLeft">
+                <p> Shop the <a href="#" class="blue">Apple Online Store</a> (1-800-MY-APPLE), visit an <a href="#" class="blue">Apple Retail Store</a>, or find a <a href="#" class="blue">reseller</a>.</p>
+                <p> Copyright © 2014 Apple Inc. All rights reserved. <a href="#" class="blue">Terms of Use</a> | <a href="#" class="blue">Privacy Policy </a></p>
+            </div>
+            <div class="footerRight">
+                <ul class="table">
+                    <li class="listItem"><a href="#" class="noStyle1"> Site Map </a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> Hot News </a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> RSS Feeds</a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> Media Info </a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> Environment </a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> Job Opportunity </a> </li>
+                    <li class="listItem"><a href="#" class="noStyle1"> Contact Us </a> </li>
+                    <li class="image"><a href="#" class="noStyle1"> 
+                        <img class="footerFlag" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/global/elements/flags/16x16/usa_2x.png" style="width: 15px;" alt="flag"> 
```

Comment:
remove style="width: 15px;" on line #89 all style should be declared in the CSS file